CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+# Contributing to SKY UX
+
+We encourage contributions from all SKY UX users. To contribute, follow this step-by-step guide. For more information about our contribution guidelines, see [SKY UX Contribution Guidelines](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines).
+
+- [Contribution Process](#contribution-process)
+- [Coding Rules](#coding-rules)
+- [Commit Message Guidelines](#commit-message-guidelines)
+- [Local Development Cheatsheet](#local-development)
+
+## <a name="contribution-process"></a> Contribution Process
+
+### File an issue
+
+Before you start any work, file an issue. If an issue already exists, comment on it and mention the `@blackbaud/skyux-maintainers` team so that we know you intend to work on it. The sooner we discuss potential code changes, the better.
```

Comment:
On the docs site we say "and mention the `@skyux-team`" in one place and "tag the SKY UX team" in a second place. ... Should those be updated to "mention the `@blackbaud/skyux-maintainers` team" like we have here?

Reply:
I just did some testing on this, and only members within the Github `blackbaud` organization can "@" mention team names. Outside contributors may only "@" mention specific user names.

When we say "mention the @skyux-team", that's specific to our Slack team (the equivalent team name on Github is "@blackbaud/skyux-maintainers") so I'm guessing we don't want to include "@skyux-team" on the public docs. Might be good to put that under a restricted-view component.

Reply:
I'll set up a PR to rework those references so that they work in the public docs.